mock/py/mockbuild/util.py

```diff
@@ -1098,6 +1098,8 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     config_opts['dynamic_buildrequires'] = True
     config_opts['dynamic_buildrequires_max_loops'] = 10
 
+    config_opts['loop_max_number'] = 12
```

Comment:
Strictly speaking, "max_number" seems like loop12 will be created .. but the max is loop11.  `dev_loop_count`?